From ab87ea4ce5d34b82a7fd539a57f99459ca2ea99e Mon Sep 17 00:00:00 2001
From: Samuel Dionne-Riel <samuel@dionne-riel.com>
Date: Thu, 11 Feb 2021 17:18:05 -0500
Subject: [PATCH] [mobile-nixos]: work around ubootTools cross-compilation
 issue

The BMP tool gets compiled for the target system, as others tools do. In
turn, this tool cannot be ran on the system doing the build.

Forcing CONFIG_*_LOGO to values fails because it has been hardcoded to
=y here.
---
 tools/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tools/Makefile b/tools/Makefile
index 2d550432ba..135fbd3863 100644
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -6,7 +6,6 @@
 # Enable all the config-independent tools
 ifneq ($(HOST_TOOLS_ALL),)
 CONFIG_ARCH_KIRKWOOD = y
-CONFIG_LCD_LOGO = y
 CONFIG_CMD_LOADS = y
 CONFIG_CMD_NET = y
 CONFIG_XWAY_SWAP_BYTES = y
-- 
2.29.2

